reports/variant_report_latest.md

```diff
@@ -32,42 +33,57 @@ Nonetheless, the apparent growth advantage of BA.2.75 over BA.5\* at whole count
 
 ![picture 5](../images/a0891368449cb6070ab832591937ace96d6b3ef704f7a3fa17e902a1c33ea035.png)
 
-An up to date version of this graph can be found here: <https://cov-spectrum.org/explore/India/AllSamples/Past3M/variants?variantQuery=NextcladePangoLineage%3ABA.5*&variantQuery1=NextcladePangoLineage%3ABA.2.75&analysisMode=CompareToBaseline&>
+An up to date version of this graph can be found [here](https://cov-spectrum.org/explore/India/AllSamples/Past3M/variants?variantQuery=NextcladePangoLineage%3ABA.5*&variantQuery1=NextcladePangoLineage%3ABA.2.75&analysisMode=CompareToBaseline&)
 
 ### Selection of other interesting lineages
 
 Many interesting BA.2 sublineages have appeared first in India. Here is a selection of some to watch. That mutations are typical of BA.2 sublineages with (potential) growth advantages, in India and also globally.
 
 #### BA.2.76 (S:R346T, S:Y248N)
 
+As of **DATE** there are approximately 1,800 sequences worldwide, and growth advantage is currently estimated at 36%. The first sample is from March.
```

Comment:
I'm picturing doing an update on these the day before the meeting. But I think it helps to give some more context to otherwise a pretty random list of variants.